test/currencyConvertor.ts

```diff
@@ -0,0 +1,73 @@
+import { ethers, waffle } from 'hardhat'
+import chai from "chai";
+import { solidity } from "ethereum-waffle";
+import CurrencyConvertorArtifact from '../artifacts/contracts/proxies/CurrencyConvertor.sol/CurrencyConvertor.json';
+import ZeroExExchangeWrapperArtifact from '../artifacts/contracts/exchange-wrappers/ZeroExExchangeWrapper.sol/ZeroExExchangeWrapper.json';
+import StarkwareArtifact from '../artifacts/contracts/exchange-wrappers/Starkware.sol/Starkware.json';
+import UsdcContractArtifact from '../artifacts/contracts/tokens/mockUsdcContract.sol/MockUsdcContract.json';
+
+const { deployContract } = waffle
+
+
+chai.use(solidity);
+const { expect } = chai;
+describe("CurrencyConvertor", () => {
+  let currencyConvertor: any;
+  let zeroExExchangeWrapper: any;
+  let starkware: any;
+  let usdc: any;
```

Comment:
nit: You should type these with `[Typechain` type annotations](https://hardhat.org/plugins/typechain-hardhat.html).